src/code/providers/lara-provider.coffee

```diff
@@ -62,6 +62,10 @@ class LaraProvider extends ProviderInterface
     else
       false
 
+  logLaraData: (laraData) ->
+    logLaraData.collaboratorUrls = @collaboratorUrls if @collaboratorUrls?.length
```

Comment:
I'm probably not understanding this right. But should this be setting the collaboratorUrls on the laraData object instead of the logLaraData object?